docs/pages/ecosystem/farcaster.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: "Open Frames - Farcaster"
+description: "An extension of Open Graph which makes link previews interactive."
+---
+
+# Farcaster
+
+Open Frames is not supported by Farcaster, but most of the Open Frames properties are directly compatible with Farcaster Frames.
```

Comment:
would phrase this differently - as there are Farcaster apps that support Open Frames in addition to Farcaster Frames. `Open Frames isn't officially supported by Farcaster, but most of the Open Frames properties are directly compatible with Farcaster Frames, and libraries for building Frames make it easy to support both conventions`